rma_purchase_analytic/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
```

Comment:
This can be deleted

Reply:
is it required to be removed? its licence line

Reply:
In the init file is not necessary, it is just the imports for the files. It does not contain actual code. It is not big deal, but you can change it, just to "standarize" with OCA